tools/lkl/cptofs.c

```diff
@@ -390,6 +409,19 @@ static int do_entry(const char *_src, const char *_dst, const char *name)
 		printf("skipping %s: unsupported entry type %d\n", src, type);
 	}
 
+	if (!ret) {
+		if (cptofs) {
+			struct lkl_timespec lkl_ts[] = { atime, mtime };
```

Comment:
Could you please add a new line here to keep checkpath happy?

Reply:
Amended.